packages/pocket-ic/tests/tests.rs

```diff
@@ -19,23 +19,22 @@ use ic_transport_types::Envelope;
 use ic_transport_types::EnvelopeContent::{Call, ReadState};
 #[cfg(not(windows))]
 use pocket_ic::common::rest::{BlockmakerConfigs, RawSubnetBlockmaker, TickConfigs};
+#[cfg(not(windows))]
```

Comment:
I'll move the unix-only tests into a separate module in a follow-up PR (to clean up this cfg mess).